ormar/fields/base.py

```diff
@@ -47,6 +48,8 @@ def __init__(self, **kwargs: Any) -> None:
         self.index: bool = kwargs.pop("index", False)
         self.unique: bool = kwargs.pop("unique", False)
         self.pydantic_only: bool = kwargs.pop("pydantic_only", False)
+        self.onupdate: Union[Callable[..., Any], Any, None] = kwargs.pop(
```

Comment:
I understand what you want and I agree - that's a nice feature and thanks. ;)

What I'm trying to say is that `ForeignKeyField` inherits from `BaseField` and defines already `onupdate` field.
So now we have one attribute in fields that is used either for FK cascades (already implemented), or for updates of the values during updates (this PR).

Simply change how the field is stored here:
```
self.on_update: Union[Callable[..., Any], Any, None] = kwargs.pop(...) # use on_update instead of onupdate name
``` 

That way we do not overwrite this in FK as this is a different functionality.